docker-images/java-base/modules/java-base/scripts/launch_java.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -x
+JAR=$1
+shift
+
+CEKIT_DYNAMIC_REOSURCES="/usr/local/dynamic-resources/dynamic_resources.sh"
```

Comment:
If this is licensed compatibly with ASL2 why don't we just copy it into strimzi and use the same thing upstream? We might chose to replace it downstream, but the point would be to have an upstream script that was directly compatible with downstream to avoid this kind of futzing around in the launch java script.